content/en/contribute/code/style-guide.md

```diff
@@ -248,6 +248,38 @@ const getResult = () =>
   result;
 ```
 
+### Implicit Returns
+
+As [noted above]({{< ref "#use-arrow-functions" >}}), implicit returns should be used for one-line arrow functions. However, for the sake of readability, they should not be used when returning a multi-line value.
+
+*Right:*
+
+```js
+const oneLineString = () => 'World';
+const oneLineObject = () => { hello: 'World' };
```

Comment:
is this supposed to return an object? if so, it should be wrapped with parenthesis  
this code is valid JS but it does not return anything, it defines a [labeled statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/label)

```suggestion
const oneLineObject = () => ({ hello: 'World' });
```

Reply:
Good catch!